packages/solid/recipes/Grid.stories.tsx

```diff
@@ -74,3 +75,26 @@ export const Example: StoryObj = {
     </Column>
   )
 };
+
+export const GridActiveRow: StoryObj = {
+  render: () => {
+    const Colors = [0xa32eacff, 0x23adccff, 0x23cc37ff, 0xcd8518ff];
+    const highlightRow = { color: 0x00ff00ff, active: { color: 0xff0000ff } };
+    const [selectedIndex, setSelectedIndex] = createSignal(0);
+    const isSelected = createSelector(selectedIndex);
+    return (
+      <Column autofocus plinko onSelect={setSelectedIndex}>
+        <For each={Colors}>
+          {(color, index) => (
+            <Row height={100} states={{ active: isSelected(index()) }} style={highlightRow}>
```

Comment:
do we want to limit the highlight row style to the active row to show the state change? looks like it's on all rows atm

![Screenshot 2024-09-23 at 11 11 08 AM](https://github.com/user-attachments/assets/4ec4ad90-e41a-4a0c-8659-3ab624282e6d)

Reply:
I'll double check this... Should be on just the active row.

Reply:
Fixed

<img width="681" alt="image" src="https://github.com/user-attachments/assets/c1f1f9da-0ed4-4b66-9ef0-76f7a2252335">